docs/content/en/docs/installation/faq.md

```diff
@@ -247,3 +247,52 @@ $ docker run -it --rm --privileged --pid=host ubuntu \
     nsenter -t 1 -m -u -n -i sh -c 'ls -la /sys/kernel/btf/vmlinux'
 -r--r--r--    1 root     root       4988627 Nov 21 20:33 /sys/kernel/btf/vmlinux
 ```
+
+### Why Tetragon fails with "operation not permitted" when loading BPF programs {#kernel-lockdown}
+
+Tetragon relies on eBPF (Extended Berkeley Packet Filter) to observe system behavior by attaching
+probes to the kernel and reading arguments, return values, and internal structures. Linux Kernel
+Lockdown is a security feature designed to prevent even the root user (UID 0) from modifying the
+kernel or reading its internal secrets. It effectively draws a line where "root" privileges stop,
+so that a compromised root account cannot compromise the kernel's integrity or exfiltrate
+cryptographic keys stored in kernel memory.
+
+Kernel Lockdown operates in two distinct modes:
+
+- **Integrity**: Prevents the user from modifying the running kernel. For example, it blocks
+  writing to `/dev/mem` or loading unsigned kernel modules.
+- **Confidentiality**: Includes all integrity protections but goes further by preventing the user
+  from reading information from the kernel that could reveal secrets.
+
+On systems with **Kernel Lockdown** enabled in `confidentiality` mode (common with Secure Boot),
+the kernel treats eBPF tracing as a potential leak of sensitive data because a BPF program could
+theoretically read secrets from kernel memory. As a result, when `lockdown=confidentiality` is
+active, the kernel blocks the `bpf()` syscall for many tracing program types and returns
+`EPERM (Operation not permitted)`. This causes Tetragon to fail when trying to load its BPF
+programs at startup.
+
+#### Checking Lockdown Status
+
+When Tetragon starts on a system with lockdown in `confidentiality` mode, it will log a warning similar to:
+
+```text
+level=warn msg="Kernel Lockdown is in 'confidentiality' mode; Tetragon will fail to load BPF programs. See https://tetragon.io/docs/installation/faq/#kernel-lockdown for details."
+```
+
+You can also check the status of the kernel lockdown by running:
+
+```shell
+cat /sys/kernel/security/lockdown
+```
+
+The output will show the current mode:
+
+* `[none]` - Lockdown is disabled (Tetragon will work)
+* `[integrity]` - Only integrity is protected (Tetragon will work)
+* `[confidentiality]` - Full lockdown (Tetragon will **NOT** work)
+
+#### Resolution Options
+
+1. **Disable Secure Boot** (in UEFI/BIOS settings) - This will disable lockdown entirely.
+2. **Use `integrity` mode instead of `confidentiality`** - Add `lockdown=integrity` to kernel boot parameters.
+3. **Use a custom kernel** without `CONFIG_SECURITY_LOCKDOWN_LSM` enabled.
```

Comment:
See above comment from Djalal but just mentioning:

I'd drop the bold statement as well also just with plain simple text
```suggestion
1. Disable Secure Boot (in UEFI/BIOS settings) - This will disable lockdown entirely.
2. Use `integrity` mode instead of `confidentiality` - Add `lockdown=integrity` to kernel boot parameters.
3. Use a custom kernel without `CONFIG_SECURITY_LOCKDOWN_LSM` enabled.
```